scripts/lib/get-routes.js

```diff
@@ -39,23 +39,17 @@ function getRoutes () {
 
     // handle some exceptions. TODO: move this into @octokit/routes
     endpoints.forEach(endpoint => {
-      // exception for uploadReleaseAssets which passes parameters as header values
-      // see https://github.com/octokit/rest.js/pull/1043
```

Comment:
I like your more generic script, but could you please keep a variation of the comment above to explain why the code is here? You could mention `uploadReleaseAssets` as en example